site/docs/concepts/README.md

```diff
@@ -1,255 +1,232 @@
-are the foundation for Flow's authorization model.
+*In this section, you'll find the most important Flow terms and concepts. If you're new to Flow, start here.*
 
-### Builds
+Data flows have three required components:
 
-Catalog entities like collections are very long-lived and may evolve over time.
-A collection's schema might be extended with new fields,
-or a transformation might be updated with a bug fix.
+* **Capture**: Flow ingests data from an outside source.
+* **Collections**: Flow maintains the captured data in cloud storage.
+* **Materialization**: Flow pushes data to an outside destination.
 
-When one or more catalog entities are updated,
-a catalog **build** validates their definitions and prepares them for execution by Flow's runtime.
-Every build is assigned a unique identifier called a **build ID**,
-and the build ID is used to reconcile which version of a catalog entity
-is being executed by the runtime.
+Optionally, they can also include:
 
-A catalog build is **activated** into Flow's runtime to deploy its captures, collections, and so on,
-possibly replacing an older build under which they had been running.
+* **Derivations**: You apply a transformation to data in a collection, resulting in a new collection.
 
-### Specifications
+All of these entities are described by a **catalog**.
 
-A catalog build begins from a set of **catalog specifications**
-which define the behavior of your catalog:
-the entities it contains, like captures, collections, and materializations,
-and their specific behaviors and configuration.
+### Catalogs
 
-You define catalog specifications using either the Flow web application,
-or by directly creating and editing YAML or JSON files which are typically managed
-in a Git repository using familiar developer workflows (often called "GitOps").
+A **catalog** comprises all the components that describe how your data flows function and behave: captures, materializations, derivations, collections, schemas, tests, and more.
+In other words, it states where data comes from and goes to, whether and how it will be transformed, and how it will be stored.
 
-These files use the extension `*.flow.yaml` or simply `flow.yaml` by convention.
-As a practical benefit, using this extension activates Flow's VS Code integration and auto-complete.
-Flow integrates with VS Code for development environment support, like auto-complete,
-tooltips, and inline documentation.
+When you configure your data flow in the web app or on the command line, you're creating a **catalog** to describe all of what you've configured.
+Catalogs are later built and deployed, creating your live data flows.
 
-Depending on your catalog, you may also have TypeScript modules,
-JSON schemas, or test fixtures which are also managed in your Git repository.
+A catalog can be represented as a graph of your data flow. At minimum, a catalog must have one capture, one collection, and one materialization.
```

Comment:
This seems also potentially misleading. For example, it's perfectly valid for a user to only ever define a materialization of a collection that someone shared with them. The bounds of a catalog aren't clearly defined in this context, which kind of makes it seem like every yaml file has to have all three of these things.

Reply:
Definitely true - thanks